bindata/bootkube/bootstrap-manifests/kube-apiserver-pod.yaml

```diff
@@ -39,6 +39,7 @@ spec:
     - --alsologtostderr
     - --v=2
     - --log-file=/var/log/bootstrap-control-plane/kube-apiserver.log
+    - --advertise-address=${HOST_IP}
```

Comment:
it seems this is not expanded, if is terraform it seems we have to use a double $$ https://github.com/hashicorp/terraform/issues/1942#issuecomment-251298610

Reply:
this is shell syntax, expanded dynamically.

Reply:
yeah, I think so, but these logs shows that is not expanding https://github.com/openshift/cluster-kube-apiserver-operator/pull/1040#issuecomment-776008908 so I assumed that the file may be preprocessed, but I´m not very familiar with this process :smile:

Reply:
kubelet is not passing the command through bash. That's why it is not expanded. I pushed another version that wraps the call with bash, similarly as in the non-bootstrap kube-apiserverr command.